cmd/data.go

```diff
@@ -230,7 +230,7 @@ func validateExistingDb(ctx context.Context, spDialect, dbURI string, adminClien
 	}
 	spannerConv := internal.MakeConv()
 	spannerConv.SpDialect = spDialect
-	err = utils.ReadSpannerSchema(ctx, spannerConv, client)
+	err = utils.ReadSpannerSchema(ctx, spannerConv, client, driver)
```

Comment:
Why is driver relevant to reading the spanner schema? This interface looks off to me.

Reply:
I have made changes to SchemaToSpannerDDL which is eventually called inside all of these functions required the driver. Unfortunately if we look at the tree, the last place where driver info is available around 5-6 functions above the one where I need it

Reply:
Please evaluate if there is a way better to make these changes without having a ripple effect change on the signatures.
We can also have a call to figure out solutions if required.